_posts/2015-05-16-prism-break.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Prism break, cuttlefish style
+---
+
+_A suggested biological color vision mechanism exploits optics_
+
+From humans to mantis shrimp, the key to seeing in color is to compare. With at least two different types of wavelength-sensitive cells in the eye, you can start to distinguish different parts of the spectrum, and thus different colored objects in the environment. The more photoreceptor types you have, the more precisely you can interpret the incoming light, depending on how finely or evenly the [photoreceptor sensitivities are spaced on the spectrum](https://arthropoda.files.wordpress.com/2010/03/human-vs-mantis.jpg). Humans are trichromatic, with three kinds of cone cells well-tuned to the short, medium, and long wavelengths of our diurnal environment. The mantis shrimp has 16 photoreceptor types, suited to both its vibrant patchwork body coloring as well as its bright surroundings. The question then is, how many photoreceptors does the cuttlefish, [a squid-like underwater chameleon](https://www.youtube.com/watch?v=SfkhEm3LfvE), have?
```

Comment:
Picking nits here, but there is a subtle difference between "different colored objects" and "different-colored objects".